packages/neuron-ui/src/locales/en.json

```diff
@@ -47,7 +47,10 @@
       "set-wallet-name": "Give your new wallet a name",
       "set-a-strong-password-to-protect-your-wallet": "Create a strong password to protect your wallet",
       "wallet-suffix": "Wallet {{suffix}}",
-      "write-down-seed": "Write down your wallet seed and save it in a safe place"
+      "write-down-seed": "Write down your wallet seed and save it in a safe place",
+      "new-name": "Use an unused name for the new wallet",
+      "complex-password": "The password is a string of 8 to 50 characters consisting of three types of characters: uppercase letters, lowercase letters, numbers, and special symbols, and must start with a letter or a number.",
+      "same-password": "Confirm the password"
```

Comment:
This message is confusing.

Reply:
Which one?

Reply:
value

Reply:
How about `The password and confirm-password should be same`

Reply:
It's ok

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/799/commits/0a1c271414b23d753b54bf8d0eaf3fe08b0fd661